geometry/geodesic-operations/README.md

```diff
@@ -1,35 +1,33 @@
-# Geodesic Operations
+# Geodesic operations
 
-Calculate the geodesic path and distance between two points.
+Calculate a geodesic path between two points and measure its distance.
 
-![](GeodesicOperations.png)
+![Image of geodesic operations](GeodesicOperations.png)
+
+## Use case
+
+A geodesic distance provides an accurate, real-world distance between two points. Visualizing flight paths between cities is a common example of a geodesic operation since the flight path between two airports takes into account the curvature of the earth, rather than following the planar path between those points, which appears as a straight line on a projected map.
 
 ## How to use the sample
 
-Click on the map to select a path destination. The geodesic path between the two points will update.
+Click anywhere on the map. A line graphic will display the geodesic line between the two points. In addition, text that indicates the geodesic distance between the two points will be updated. Click elsewhere and a new line will be created.
 
 ## How it works
 
-To create a geodesic path between two points:
-
-1. Create a `Polyline` using two points.
-2. Pass this polyline to: `GeometryEngine.densifyGeodetic(polyline, segmentLength, unitOfMeasurement, GeodeticCurveType.GEODESIC)`. This will create a new polyline with segments of length `segmentLength` and `LinearUnit` set to `unitOfMeasurement`. The curve will be geodesic.
-3. You can set this geometry to a `Graphic` to display the curve in a `GraphicsOverlay`.
-4. To get the distance, use `GeometryEngine.lengthGeodetic(pathGeometry, unitOfMeasurement, GeodeticCurveType.GEODESIC)`
+1. Create a `Point` in New York City and display it as a `Graphic`.
```

Comment:
I made a comment here for Android too that we don't need to be so specific about where the point is: we removed the reference to NYC.

Reply:
That's a good point, I also found that a bit strange. It's now removed.